service/pom.xml

```diff
@@ -22,9 +22,14 @@
         <version.curator>5.2.0</version.curator>
         <version.curator.test>5.2.0</version.curator.test>
         <version.datawave.accumulo-api>4.0.0</version.datawave.accumulo-api>
+        <version.datawave.accumulo-api>4.0.0</version.datawave.accumulo-api>
+        <version.datawave.audit-api>4.0.1</version.datawave.audit-api>
         <version.datawave.starter-datawave>4.0.2</version.datawave.starter-datawave>
         <version.datawave.starter-datawave-audit>4.0.2</version.datawave.starter-datawave-audit>
         <version.hadoop>3.3.4</version.hadoop>
```

Comment:
Maybe update hadoop as you did in other prs for this task

Reply:
up to 3.3.5